bleachbit/Action.py

```diff
@@ -408,15 +409,13 @@ def run_process():
                     rc = 0  # unknown because we don't wait
                     from subprocess import Popen
                     Popen(self.cmd)
-            except Exception, e:
+            except Exception as e:
                 raise RuntimeError(
                     'Exception in external command\nCommand: %s\nError: %s' % (self.cmd, str(e)))
             else:
                 if not 0 == rc:
-                    logger = logging.getLogger(__name__)
```

Comment:
I remember reading somewhere that defining the logger locally is better than getting it from the module, so I will need to check into this change.